### PR TITLE
[ENHANCEMENT] Add official support for Node.js v20

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: tests/unit/tasks/server/livereload-server-test.js [hunk 1]
```diff
@@ -100,7 +100,9 @@ describe('livereload-server', function () {
           port: 4200,
         })
         .then(function () {
-          http.get('http://localhost:4200/_lr/livereload.js', function (response) {
+          let httpGetOptions = { headers: { Connection: 'close' } };
+
+          http.get('http://localhost:4200/_lr/livereload.js', httpGetOptions, function (response) {
```
Comment (bertdeblock) [inline tests/unit/tasks/server/livereload-server-test.js:105]: I'm unsure if this is the right fix, but on Node.js v20, this `GET` request is being sent with `Connection: keep-alive` as opposed to `Connection: close` on older Node.js versions. `Connection: keep-alive` causes the `done` callback below to not being called anymore.

Comment (NullVoxPopuli) [reply tests/unit/tasks/server/livereload-server-test.js:105]: was there ever a problem with `ember test`? or was it just the tests here?

**Review by NullVoxPopuli — APPROVED** [review]

File: tests/unit/tasks/server/livereload-server-test.js (hunk 1 above)
Comment (bertdeblock) [reply tests/unit/tasks/server/livereload-server-test.js:105]: I assume it's just this test. I think if `ember test` or any other command didn't work on Node.js v20, we would've gotten more bug reports already. Just a guess though!

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**